### DISCUSSION TITLE
Microfrontends -Docker based production deployment & manage configurations

### DISCUSSION THREAD

**Discussion by raolak** [web-infra-dev/modern.js#6048] [General]
have main /host app and sub apps workfing well though module federation. But I now need to prod deploy these apps using docker. 

Please help me if you have any reference. Really appreciate. I am primarily  looking at how modernjs. ip configurations can be configured for production usecases

Thanks!

**Comment by zllkjc** [comment]
Modern.js will not configure your IP, it depends on where your docker is running.

**Comment by zllkjc** [comment]
you can refer to https://modernjs.dev/en/guides/basic-features/deploy.html#nodejs to deploy your app on docker which Including Node.js environment.

**Comment by raolak** [comment]
Thanks for the suggestion. I referred the document and followed the steps.

I followed these steps without docker
host  (localhost:8080)
npx modern deploy
node .output/index


Followed the same above step as that of the host. Host modern.config.js has
sub app ip configured and sub apps starts in their respective ports
Sub app 1 - (localhost: 8081)
Sub app 2 - (localhost: 8082)

But I see blank pages on the host...

While if i run my apps using 'pnpm run dev' (host and sub apps), it all
works well.

I will be happy to connect and walk you through my app. I am almost
production ready , but blocked on this production deployment.

Thank you !
